Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Storage/Queries/Ride.hs

```diff
@@ -683,13 +686,17 @@ instance FromTType' BeamR.Ride Ride where
     let toLocMap = maximumBy (comparing (.order)) toLocationMappings
     toLocation <- QL.findById toLocMap.locationId >>= fromMaybeM (InternalError $ "ToLocation not found in ride for toLocationId: " <> toLocMap.locationId.getId)
     tUrl <- parseBaseUrl trackingUrl
+    booking <- QB.findById (Id bookingId) >>= fromMaybeM (BookingNotFound bookingId)
```

Comment:
Why to search for booking? MerchantId is already there in ride table. If it is Nothing then only search for booking.

Reply:
Used default city